### PR TITLE
add link to slack-signup

### PR DESCRIPTION
**Pull request by usefulthink**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -234,6 +234,7 @@
 				<a href="https://stackoverflow.com/questions/tagged/three.js">questions</a><br />
 				<a href="https://discourse.threejs.org/">forum</a><br />
 				<a href="http://webchat.freenode.net/?channels=three.js">chat</a><br />
```
Comment (Fasani) [inline index.html]: Might be also helpful to change this link to "irc" rather than "chat".

**Review by Fasani — CHANGES_REQUESTED** [review]

**Review by Fasani — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by looeee** [comment]
Shouldn't the slack link be https://threejs.slack.com ?

**Comment by Fasani** [comment]
@looeee that is the slack url but @usefulthink has created a small self invite app. You can’t get into slack without an invite and the app hosted on heroku allows you to invite yourself. It’s a common practice. Give it a try and I will see you on the inside! :-)

**Comment by looeee** [comment]
OK, that makes sense 😄

**Pull request merged**